src/api/app/models/status_message.rb

```diff
@@ -1,12 +1,49 @@
 class StatusMessage < ApplicationRecord
-  belongs_to :user
+  belongs_to :user # TODO: rename as creator
+  has_many :status_message_acknowledgements, dependent: :destroy
+  has_many :users, through: :status_message_acknowledgements
+
   validates :user, :severity, :message, presence: true
+
   scope :alive, -> { where(deleted_at: nil).order('created_at DESC') }
+  scope :announcements, -> { alive.where(severity: 'announcement') }
+
+  enum severity: { information: 0, green: 1, yellow: 2, red: 3, announcement: 4 }
```

Comment:
Green, yellow and red aren't severities, they are colors. I know it was already like this, but if we're refactoring status messages, we should change this. Yellow could be Warning, Green is just like Information (we don't need it), Red is for issues/problems, Announcement is the same as information.

Reply:
Totally agree. @hennevogel what do you think?

Reply:
I also agree but I don't think it's time to do it. I wouldn't say we are refactoring Status Message but adding extra functionality. And since we have to keep backward compatibility it won't be fast.

We have caught many things that can be improved regarding status message code and we wrote them down for the future. Things like: renaming severity, moving status message from `main` to its own namespace, renaming `user_id` to `creator_id`, etc.
But I believe none of them are our goal now.

Reply:
@saraycp Aright no worries. It's true that it's out of scope. I'm fine with this.